wpeverest/user-registration
# UR 1071 Dev -Elementor widget

### All Submissions:






### Changes proposed in this Pull Request:



Closes # Previously we only had a registration form in elementor wigets this PR introduce other forms as weell

### How to test the changes in this Pull Request:
 Note : All user-registration blocks should be shown as widgets in elementor except content-restriction 
1.`activate` elementor and make or edit page with elementor
2.under `user registration` section you can see all widgets
3.Verify if all widgets are working preperly

### Types of changes:




### Other information:




### Changelog entry

>  Dev -Elementor widget
